scripts/makeLegacyPlots.sh

```diff
@@ -77,83 +79,69 @@ echo $obj1,$obj2
+#python scripts/$plotter --dir $basedir/$tag/$channel --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel6 --channel $channel --tag $tag --label "DNN bin number" --quit --ratio --log --blind-range 0.0 0.5 --no-binwidth --binNXaxis --doStatSystBand --removeESsystBand
+#python scripts/$plotter --dir $basedir/$tag/$channel --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel7 --channel $channel --tag $tag --label "DNN bin number" --quit --ratio --log --blind-range 0.0 0.5 --no-binwidth --binNXaxis --doStatSystBand --removeESsystBand
+#python scripts/$plotter --dir $basedir/$tag/$channel --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel8 --channel $channel --tag $tag --label "DNN bin number" --quit --ratio --log --blind-range 0.0 0.5 --no-binwidth --binNXaxis --doStatSystBand --removeESsystBand
+#
+#
+## PLOTS WITH BIN-NUMBER AS X-AXIS AND NO SYST BAND
+#python scripts/$plotter --dir $basedir/$tag/$channel --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel1 --channel $channel --tag $tag --label "DNN bin number" --quit --ratio --log --blind-range 0.5 1 --no-binwidth --binNXaxis
+#python scripts/$plotter --dir $basedir/$tag/$channel --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel2 --channel $channel --tag $tag --label "DNN bin number" --quit --ratio --log --blind-range 0.5 1 --no-binwidth --binNXaxis
+#python scripts/$plotter --dir $basedir/$tag/$channel --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel3 --channel $channel --tag $tag --label "DNN bin number" --quit --ratio --log --blind-range 0.5 1 --no-binwidth --binNXaxis
+#python scripts/$plotter --dir $basedir/$tag/$channel --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel4 --channel $channel --tag $tag --label "DNN bin number" --quit --ratio --log --blind-range 0.5 1 --no-binwidth --binNXaxis
+#python scripts/$plotter --dir $basedir/$tag/$channel --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel5 --channel $channel --tag $tag --label "DNN bin number" --quit --ratio --log --blind-range 0.5 1 --no-binwidth --binNXaxis
+#python scripts/$plotter --dir $basedir/$tag/$channel --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel6 --channel $channel --tag $tag --label "DNN bin number" --quit --ratio --log --blind-range 0.0 0.5 --no-binwidth --binNXaxis
+#python scripts/$plotter --dir $basedir/$tag/$channel --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel7 --channel $channel --tag $tag --label "DNN bin number" --quit --ratio --log --blind-range 0.0 0.5 --no-binwidth --binNXaxis
+#python scripts/$plotter --dir $basedir/$tag/$channel --year $year --var DNNoutSM_kl_1 --reg $reg --sel $sel8 --channel $channel --tag $tag --label "DNN bin number" --quit --ratio --log --blind-range 0.0 0.5 --no-binwidth --binNXaxis
```

Comment:
Instead of commentig all these lines you can delete them and only leave the 1st one of each section as an example.